lms/static/sass/discussion/elements/_editor.scss

```diff
@@ -22,6 +22,11 @@
 
 .wmd-preview-container {
   @include discussion-new-post-wmd-preview-container;
+
+  .discussion-board &,
```

Comment:
Nit: I don't like having rules for container elements added to children. This seems like a hack, and we'll have to add more such hacks in the future when we show the discussion in other places. Is there any chance that the rule can be made more generally so that it isn't specific for particular containers.